### ISSUE TITLE
Document Forced Export dependency

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/NetFixClient#4]
Can we include Forced Export as a submodule or would it be better to create a readme with where to grab Forced Export from for inclusion?

**Comment by DanRStevens** [comment]
I started some work on porting ForcedExports from SVN to Git (and renaming it to OP2Internal). There were a few complications with messy history, which I never quite dealt with. Ideally, I should get that done, so we could add it as a submodule.

**Comment by Brett208** [comment]
I think the rest of the issues will have to wait on Forced Exports then. I wouldn't be able to compile and check updates until we have it figured out.

For a quick hitter, I had thought about trying to fix the log file location bug for the next release of Outpost 2, but maybe there is too much work.

**Comment by Brett208** [comment]
I updated ForcedExport to use the v141 toolset (VS17). I had to update a couple of project settings like the release build targeting an executable and the MD vs MT settings. Recommend deleting the files ForcedExports.dsp and ForcedExports.dsw and replacing ForcedExports.vcxproj with the attached version.

[ForcedExports.zip](https://github.com/OutpostUniverse/NetFixClient/files/2648857/ForcedExports.zip)

**Comment by DanRStevens** [comment]
This is much appreciated. I agree about deleting the old project files. That can be done in a branch, with the new project file added.

We'll need to sort out the dependency issue before we can complete the project file settings. Effectively, this is blocked until I get the SVN -> Git conversion done.

**Comment by Brett208** [comment]
While OP2Internal (ForcedExports) needs more work, it is now 'documented' as a submodule. I am going to close this issue. We can open a new issue targeting specific issues with OP2Internal.
